configfiles/1033_preprocess_snort.conf

```diff
@@ -7,7 +7,10 @@ filter {
   if [type] == "snort" {
     # This is the initial parsing of the log
     grok {
-      match => ["message", "\[%{INT:gid}:%{INT:sid}:%{INT:rev}\]\s%{DATA:alert}\[Classification:\s+%{DATA:classification}\]\s+\[Priority:\s+%{INT:priority}\]:\s+{%{DATA:protocol}}\s+%{IPV4:source_ip}:%{INT:source_port}\s+->\s+%{IPV4:destination_ip}:%{INT:destination_port}%{GREEDYDATA:message}", "message", "\[%{INT:gid}:%{INT:sid}:%{INT:rev}\]\s%{DATA:alert}\[Classification:\s+%{DATA:classification}\]\s+\[Priority:\s+%{INT:priority}\]:\s+{%{DATA:protocol}}\s%{IPV4:source_ip}\s+->\s+%{IPV4:destination_ip}"]
+      match => ["message", "\[%{INT:gid}:%{INT:sid}:%{INT:rev}\]\s%{DATA:alert}\[Classification:\s+%{DATA:classification}\]\s+\[Priority:\s+%{INT:priority}\]:\s+<%{DATA:interface}>\s+{%{DATA:protocol}}\s+%{IPV4:source_ip}:%{INT:source_port}\s+->\s+%{IPV4:destination_ip}:%{INT:destination_port}",
```

Comment:
Thanks for the PR!  Help me understand what's going on here.  If I'm reading it correctly, originally there were 2 matches for 2 different formats and this PR is doubling that to 4 to account for interface or no interface.  Good so far?  If that's the case, should the 1st match have `%{GREEDYDATA:message}"` at the end (just like the 3rd match does)?